adapters/smarthub/params_test.go

```diff
@@ -35,11 +35,9 @@ var invalidParams = []string{
 	`{"partnerName":"partnertest"}`,
 	`{"seat":"9Q20EdGxzgWdfPYShScl"}`,
 	`{"token":"Y9Evrh40ejsrCR4EtidUt1cSxhJsz8X1"}`,
-	`{"seat":"9Q20EdGxzgWdfPYShScl", "token":"alNYtemWggraDVbhJrsOs9pXc3Eld32E"}`,
```

Comment:
Since none of your params are now considered required, please add the following test cases to `validParams` to ensure that the absence of each param is valid, the empty string is allowed for `partnerId` and a minimum length of 1 is valid for `seat` and `token`:
```
{"partnerName":""}
{"partnerName":"1"}
{"seat":"1"},
{"token":"1"},
```

Reply:
ok I added this test cases and all tests was passed

Reply:
> Since none of your params are now considered required

@bsardo  - The seat and token params are still required in this schema.

Reply:
Hmm this cases I added to notValidParams because we using  parnterName+seat+token OR seat+token in aliases, this means that the pair seat+token will always be required.

Reply:
> > Since none of your params are now considered required
> 
> @bsardo - The seat and token params are still required in this schema.

Oops not sure what I was looking at when I reviewed this 😄

Reply:
@SmartHubSolutions can you please add the following tests to `validParams` to verify what is required and minimum lengths:
```
`{"partnerName":"", "seat":"1", "token":"1"}`,
`{"seat":"1", "token":"1"}`,
```

Reply:
no problem I addeed them, tests passed